readme.md

```
@@ -15,120 +15,346 @@
+🔗 [`docs/architecture.md`](https://github.com/ArnauAregall/aregall-agenthub/blob/main/docs/architecture.md)
+🔗 [`docs/openapi.yaml`](https://github.com/ArnauAregall/aregall-agenthub/blob/main/docs/openapi.yaml)
 
 ---
 
 ## 5. Historias de Usuario
 
-> Documenta 3 de las historias de usuario principales utilizadas durante el desarrollo, teniendo en cuenta las buenas prácticas de producto al respecto.
+El backlog completo de AgentHub comprende 18 historias de usuario distribuidas en 5 fases de entrega (Phase 0 a Phase
+5), gestionadas en el [GitHub Projects board](https://github.com/users/ArnauAregall/projects/1) del repositorio privado
+y documentadas en [
+`docs/backlog/user-stories.md`](https://github.com/ArnauAregall/aregall-agenthub/blob/main/docs/backlog/user-stories.md).
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside link text

(MD039, no-space-in-links)